site-src/guides/tls.md

```diff
@@ -98,6 +101,55 @@ would be invalid.
 {% include 'standard/tls-cert-cross-namespace.yaml' %}
 ```
 
+## Upstream TLS
+
+Upstream TLS settings are configured using the experimental `BackendTLSPolicy`
+applied to a `Service` that accesses the backend to be targeted.
+This type of TLS is used when the backend pod has its own certificate and the
+application developer needs to convey how to successfully connect to the
+backend pod from the Gateway.
+
+### TargetRefs and TLS
+
+The `TargetRef` identifies the `Service` for which your HTTPRoute requires TLS.
+The `TLS` configuration for upstream TLS contains a required `Hostname`, and either
+`CACertRefs` or `WellKnownCACerts`.
+`Hostname` refers to the SNI the Gateway should use to connect to the backend, and
+must match the certificate served by the backend pod.
+`CACertRefs` refer to one or more PEM-encoded TLS certificates.
+If there are no specific certificates to use, then you must set `WellKnownCACerts` to
+"System" to tell the Gateway to use a set of trusted CA Certificates. There may be some
+variation in which system certificates are used by each implementation. Refer to documentation
+from your implementation of choice for more information.
```

Comment:
Would recommend adding some newlines here to split this into more than one paragraph. The deploy preview merges this all into one paragraph that can be a bit hard to follow: https://deploy-preview-2493--kubernetes-sigs-gateway-api.netlify.app/guides/tls/#targetrefs-and-tls